lib.php

```diff
@@ -653,16 +653,11 @@ function block_coursefeedback_get_editerrors($feedbackid) {
 function block_coursefeedback_set_active($feedbackid) {
```

Comment:
Ich würde wenigstens einen kurzen Satz dazu in den Docstring schreiben, was die Funtktion tut. Ich meine in erster Linie, dass Sie eben nicht _nur_ ggf. die Config setzt, sondern eben auch immer das "Vergessen" herbeiführt, ob je ein User das alte Feedback beantwortet hat.
Und in `doesn"t` sollte ein einfaches Anführungszeichen `'`  stehen.

Reply:
Bei den Docs wäre ich jetzt nicht päpstlicher als der Papst. Die Funktion ist ja schon sehr spezifisch benannt und am Ende müsste man dann auch alle Funktionen einmal anfassen. Solange nicht nebenbei noch andere Sachen als "set active" gemacht werden reicht die Doc. Den Rechtschreibfehler sollte man richtig stellen.

Reply:
Habe eine Beschreibung hinzugefügt [e81bbe5](https://github.com/eberhardt/moodle-block_coursefeedback/pull/31/commits/e81bbe53a99bc814bb76d2f00a179ef12d364d74)
und merge damit den PR.